var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -6,7 +6,7 @@
 from spack import *
 
 
-class SuperluDist(CMakePackage, CudaPackage):
+class SuperluDist(CMakePackage, CudaPackage, ROCmPackage):
```

Comment:
Is this related to this PR?

Reply:
Yes, this is related. Superlu-dist is used as one example GPTune can autotune, where we tuned its GPU performance on both AMD and NVIDIA GPUs.

Reply:
I changed  
if '+rocm' in spec:
to 
if '+rocm' in spec and spec.satisfies('@amd'):

when adding rocm related cmake flags. So if it's not the version "amd", +rocm has no effect. 

I don't know if there is a better way to handle this.